tasks/fine_tuning_sbert/notebooks/Fine_tuning_SBERT_with_callback.ipynb

```diff
@@ -1,3160 +1,2839 @@
-      "text/plain": [
-       "<Figure size 576x432 with 2 Axes>"
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AlL8L6HzjeWr"
+      },
+      "source": [
+        "# Data Loading"
       ]
-     },
-     "metadata": {
-      "tags": []
-     },
-     "output_type": "display_data"
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "============================================================\n",
-      "Current Score is: 0.9444444444444444\n",
-      "Current ACC_LIST is: [0.8148148148148148, 0.9444444444444444]\n",
-      "Epoch: 1\n",
-      "Best score: 0.9444444444444444\n",
-      "============================================================\n"
-     ]
+      "cell_type": "code",
+      "metadata": {
+        "id": "jm-fbjB9Rkzx"
+      },
+      "source": [
+        "\"\"\"\n",
+        "MAKE SURE THAT THIS IS ALWAYS YOUR INDIVIDUAL NUMBER (FOR REASONS TO NOT OVERWRITE DATA)\n",
+        "\"\"\"\n",
+        "\n",
+        "GLOBAL_EXPERIMENT_NUMBER = 16"
```

Comment:
If this is not going to matter in the future, let's join this cell with the next one and make it a random number. We'll delete the folder at the end anyway right?

Reply:
We do delete the folder yes. But I'm concerned if 2 people running the notebook at the same time will get the same random_number, then both run's models will be saved in the same place. If it's a low probability, we can totally make it a random number.

I merged the cells too, they didn't need to be separate.

Reply:
Also shifted the cell to remove the folder a little above so it's not missed while running everything